ui/package.json

```
@@ -26,7 +26,8 @@
     "react-icons": "^4.12.0",
     "react-modal": "^3.16.1",
     "react-query": "^3.39.2",
-    "typescript": "4.9.4"
+    "typescript": "4.9.4",
+    "swiper": "^9.4.1"
```

This version is quite old. Any reason not to use the most current version?
```suggestion
    "swiper": "^11.1.4",
    "typescript": "4.9.4",
```